src/smart-components/MessageSearch/context/hooks/useGetSearchedMessages.ts

```diff
@@ -47,11 +47,11 @@ function useGetSearchedMessages(
     if (sdk && channelUrl && sdk.createMessageSearchQuery && currentChannel) {
       if (requestString) {
         const inputSearchMessageQueryObject: MessageSearchQueryParams = {
-          ...messageSearchQuery,
           order: MessageSearchOrder.TIMESTAMP,
           channelUrl,
```

Comment:
Do we really want to override? I think we explicitly didnt wanted to do this before

Anyways, please add comment with ticket number before merge

Reply:
I confirmed the other platforms are allowing to override `messageSearchQuery`.